### ISSUE TITLE
TypeError: Cannot read property 'rawRows' of undefined

### ISSUE THREAD

**Issue by blumk** [OP-Engineering/op-sqlite#152]
Hi 👋 I've upgraded the library from `7.3.0` to `8.0.1` and changed the database execution call from `db.executeAsync` to `db.execute`. 

Running into the following error: `[op-sqlite] SQL execution error: not an error`

Then I've wrapped the following line in `enhanceDd` to a try/catch:
`let intermediateResult = await db.execute(query, sanitizedParams);`

which fails with: `TypeError: Cannot read property 'rawRows' of undefined`

Query I'm trying to run:
```
      CREATE TEMP TABLE temp_localDate AS
      SELECT
        startTimestamp,
        DATETIME(
          startTimestamp,
          'localTime'
        ) AS localStartDate,
        (strftime('%s', endTimestamp) - strftime('%s', startTimestamp)) AS duration
      FROM point_track;
```

Happy to provide more details.

**Comment by ospfranco** [comment]
Latest version is 8.0.3, try that one

**Comment by blumk** [comment]
Thanks for the bug fix, `8.0.3` addresses the issue.

Feel free to publish the release on GitHub:

<img width="194" alt="image" src="https://github.com/user-attachments/assets/9392bbc0-3fa0-40da-b44e-d1a265b32e01">

**Comment by ospfranco** [comment]
You can just hover over the dependency in vscode and it will show you the latest published version

![OSP 000377](https://github.com/user-attachments/assets/df9bc377-2644-4877-9ef6-8305f62eda4e)
